proto/Makefile

```diff
@@ -16,17 +16,15 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PWD=$(shell pwd)
-
-PROTOC      := $(PWD)/../../../../../bin/protoc
+PROTOC      := $(GOPATH)/bin/protoc
```

Comment:
In shell (err, bash) you can easily strip off everything after the first entry: `${GOPATH%%:*}`. Probably something similar can be done in make.